api/src/accounts.rs

```diff
@@ -61,21 +60,6 @@ pub fn get_account_resources(context: Context) -> BoxedFilter<(impl Reply,)> {
         .boxed()
 }
 
-// GET /accounts/<address>/resource/<resource_type>
-pub fn get_account_resource(context: Context) -> BoxedFilter<(impl Reply,)> {
```

Comment:
@msmouse - Maybe I am missing something but this API path is still under /accounts - so it makes sense to keep this under accounts.rs?

Reply:
See events.rs -- they are group by functionality or the class powering the endpoints (`struct Events` or `struct State` in this case), not by the path.